bin/get_metadata.py

```diff
@@ -42,6 +43,14 @@ def find(self, d, tag):
                     for i in v:
                         yield from self.find(i, tag)
 
+    def read_run_info(self):
```

Comment:
I think it would be good to add a unit test for this function. In case a bug is introduced it will make it easier to pin down where the fault originates from.